lib/libc/stdlib/malloc/mrs/mrs.c

```diff
@@ -935,21 +939,26 @@ print_cheri_revoke_stats(char *what, struct cheri_revoke_syscall_info *crsi,
 }
 #endif /* PRINT_CAPREVOKE */
 
-static inline void
-slab_free_next_descriptor(struct mrs_descriptor_slab *s)
+static inline void *
+slab_pop_descriptor(struct mrs_descriptor_slab *s)
```

Comment:
Why "pop"? These entries are FIFO, aren't they?

Reply:
I've switched to "dequeue" which doesn't imply a stack. I'd wanted to avoid "next" since it's an action.